themes/default/content/blog/announcing-kubernetes-helm-release-resource-preview/index.md

```diff
@@ -0,0 +1,305 @@
+---
+title: Announcing Preview for Helm Release Resource for Kubernetes
+h1: "Announcing Preview for Helm Release Resource for Kubernetes"
+date: "2021-09-07"
+meta_desc: "Today we are announcing preview of a new Helm Release resource for Kubernetes which adds support for Helm natively on Pulumi."
+meta_image: "helm-release.gif"
+authors: ["vivek-lakshmanan"]
+tags: ["Kubernetes", "helm"]
+---
+
+Kubernetes has been a significant focus of Pulumi since its very beginnings. Pulumi added support for installing [Helm charts](https://helm.sh/) way back in [2018](https://www.pulumi.com/blog/using-helm-and-pulumi-to-define-cloud-native-infrastructure-as-code/) and it has seen significant adoption by users since. However, Pulumi's current Chart integration lacks support for some increasingly common advanced features in Helm charts, e.g.:
+
+1. Support for [Helm lifecycle hooks](https://helm.sh/docs/topics/charts_hooks/)
+2. Handling sub-charts and [dependencies](https://helm.sh/docs/topics/charts/#chart-dependencies)
+
+These and other limitations have resulted in Helm support becoming a common painpoint for our Kubernetes users.
+
+Today we are excited to announce the *preview* of a new [Helm Release](https://www.pulumi.com/docs/reference/pkg/kubernetes/helm/v3/release/) resource starting with [v3.7.0](https://github.com/pulumi/pulumi-kubernetes/releases/tag/v3.7.0) of the Pulumi Kubernetes Provider and SDK in all Pulumi supported languages. Note this is not meant to signal deprecation of the [Chart](https://www.pulumi.com/docs/reference/pkg/kubernetes/helm/v3/chart/) resource which will continue to be supported in parallel. This new resource provides Pulumi users more options to choose the right tool for their use-case. The rest of this blog post will highlight how this resource differs from the existing Chart component resource and describe how and when to use the new resource.
+
+<!--more-->
+
+## How is Helm Release resource different?
+
+Pulumi's Helm Chart integration was implemented as a [Component Resource](https://www.pulumi.com/docs/intro/concepts/resources/#components) which effectively extracts the corresponding Kubernetes resources' manifests from the chart and installs them as if they were individually specified in the Pulumi user program. This allowed users to install community developed Helm charts, while providing fine-grained control over the respective Kubernetes resources installed in the process. For instance, the installed resources are subject to all policies specified through Policy-as-Code policy packs registered with Pulumi. Other users have used Pulumi's powerful transformation support to manipulate Helm charts to fit their desired needs before installing them in their environment.
+
+In contrast, the new Helm Release resource is a thin wrapper around Helm itself. This provides the following advantages:
```

Comment:
```suggestion
In contrast, the new Helm Release resource uses the Helm SDK, bringing the full feature set of Helm directly to Pulumi users. This provides the following advantages:
```

Trying to avoid using "wrapper" as a description, it has an implication of using the Helm binary.

Reply:
Makes sense! Fixed.